bnd-workspace/de.fraunhofer.abm.app/src/de/fraunhofer/abm/app/controllers/UserApprovalController.java

```diff
@@ -46,6 +54,14 @@ public String getApproval(RESTRequest rr) {
 			user.getCredentials().put("password", password);
 			Group registeredUserGroup = (Group) userAdmin.getRole("RegisteredUser");
 			registeredUserGroup.addMember(user);
+			/**String sbj = params.get("username") + " Approved on ABM";
```

Comment:
Why are those changes commented out? If they are part of a later pull request, they don't need to be submitted in this one. Please remove.